app/helpers/timelines_helper.rb

```diff
@@ -307,25 +307,21 @@ def internationalized_columns_select(collection)
 
   include Gon::GonHelpers
 
-  def visible_timeline_paths(_visible_timelines = [])
-    @visible_timelines.inject({}) do |timeline_paths, timeline|
-      timeline_paths.merge(timeline.id => { path: project_timeline_path(@project, timeline) })
-    end
-  end
-
-  def push_visible_timeline_paths(visible_timelines)
-    gon.timelines = visible_timeline_paths visible_timelines
+  def push_visible_timelines(visible_timelines, target = gon)
+    target.timelines = visible_timelines.map { |timeline|
```

Comment:
Assuming that you are doing a refactoring in this file:

Did you notice, that you changed the behaviour here?
Previously the `_visible_timelines` parameter was ignored and the instance variable `@visible_timelines` was taken instead.

Your new code really cares for the parameter it receives.

While this sounds like a sensible move, I want to make sure it was an intended one...